src/igvfd/types/donor.py

```diff
@@ -26,7 +26,7 @@ class Donor(Item):
         Path('award', include=['@id', 'component']),
         Path('lab', include=['@id', 'title']),
         Path('submitted_by', include=['@id', 'title']),
-        Path('phenotypic_features.feature', include=['@id', 'feature', 'term_name'])
+        Path('phenotypic_features.feature', include=['@id', 'feature', 'term_name', 'amount', 'amount_units'])
```

Comment:
I am not sure where the amount and amount units coming from?

Reply:
you're right this should have been quantity & quantity_units. My mistake.